### ISSUE TITLE
Fedora: Use subpackages instead of 3 spec files

### ISSUE THREAD

**Issue by fhaverkamp** [ibm-genwqe/genwqe-user#47]
We got the following suggestion from the Fedora team:
Including 3 spec files in one tarball in not necessary. It is much better to use a single spec file with subpackages.

**Comment by krisman** [comment]
This is fixed by my merge request.  We can close as soon as my patches are merged upstream.

**Comment by fhaverkamp** [comment]
Gabriel contributed this set of patches, which should address the requested package build simplifications: https://github.com/ibm-genwqe/genwqe-user/commit/639133f6a012f9ae77005091ed8baa94af72eb84.

I tried myself and fixed some more things. There was the rm of the non-existent version.mk file, which I removed entirely. The rpmbuild rule which creates the tar.gz is now overwriting the version.mk in any case. We do not copy the .git directory into the tar.gz anymore. Furthermore I do not move binaries in the lib directory. I have seen that the install directories seem to be according to my internal proposal already.

What is still strange is a lot of debug info related stuff in the resulting zlib rpm. Once you are fine with the rpms, we should use the time to review their contents and the locations carefully, such that they match what we like to see there. Docu needs to be changed later on too. I also tend to believe that we should change the install default directories instead of trying to fix things up in the spec file. For this we need to fixup the instdir in the Makefiles (tools/lib/...).

I think we are on the right path ...
